src/metaopt/tests/mongodb_test.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from metaopt.io.database.mongodb import MongoDB
+
+class MongoDB_test():
```

Comment:
Why not using unittest.TestCase and tests directly inside the class rather than gluing them together in a function later?

Reply:
Gotcha I adjusted this per assert in the functions inside the class

Reply:
No need for subclassing in general: https://docs.pytest.org/en/latest/getting-started.html

Reply:
No but the assert methods coming with unittest.TestCase are quite useful. :slightly_smiling_face: